stable/sonatype-nexus/requirements.yaml

```diff
@@ -0,0 +1,5 @@
+dependencies:
```

Comment:
I don't think we should add cert-manager as a dependency. Imagine we'd do this with any chart that could expose a service. This should be installed separately and usually not on a per-app basis.